### ISSUE TITLE
Invisible font in message preview with image or audio in Activity Center in dark mode

### ISSUE THREAD

**Issue by qoqobolo** [status-im/status-react#12274]
# Bug Report

## Problem 
Previews for audio and image messages in Activity center have black color in dark mode, making it unreadable. 

#### Actual behavior

<img width="464" alt="Screenshot 2021-07-05 at 12 27 49" src="https://user-images.githubusercontent.com/67952253/124457732-6b486a80-dd8c-11eb-8606-682ccca03979.png">

### Reproduction

Device1:
- Have Status in dark mode
Device2:
- Start 1-1 chat with Device1
- Send an image or audio
Device1:
- Open Activity center

### Additional Information

- Status version: nightly 05/07
- Operating System: Android, iOS

@briansztamfater presumably it's a regression after #12227 . Take a look when you have time, please.
This is not a critical issue but would be nice to have it fixed.

**Comment by qoqobolo** [comment]
Fixed in https://github.com/status-im/status-react/pull/12266
